homework-1/task_1.py

```diff
@@ -0,0 +1,75 @@
+"""
+
+TASK 1
+
+Write a class to represent a Cash Register.
+You class should keep the state of price total and purchased items
+
+Below is a starter code:
+------------------------
+1. you can add new variables and function if you want to
+2. you will NEED to add accepted method parameters where required.
+For example, method add_item probably accepts some kind of an item?..
+3. You will need to write some examples of how your code works.
+
+"""
+
+
+class CashRegister:
+
+    def __init__(self):
+
+        self.total_items = []
+        self.total_price = 0
+        self.discount = 0
+
+    def add_item(self, item):
+        self.total_items.append(item)
+        print(f"You have added {item} to the shopping basket.")
+
+    def remove_item(self, item):
+        self.total_items.remove(item)
```

Comment:
Would have perhaps been nice to think about what would happen if someone tried to remove a non-existent item here. Would also have been nice to be able to remove by name only rather than by name and price, but that's only a small point